### PR TITLE
feat: sdk and middleware module

### PR DESCRIPTION
**Pull request by maxiroellplenty**
## Why:

Closes: #ID

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/nuxt.config.ts [hunk 1]
```diff
@@ -77,6 +77,7 @@ export default defineNuxtConfig({
     },
   },
   modules: [
+    '@vue-storefront/nuxt',
```
Comment (ksted) [inline apps/web/nuxt.config.ts]: Please add this module to the bottom of the list instead. The way the list is arranged groups modules according to how they fit into the Nuxt module ecosystem (official, community, third-party).

File: apps/web/composables/useActiveShippingCountries/useActiveShippingCountries.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import type { ActiveShippingCountry } from '@plentymarkets/shop-api';
-import { useSdk } from '~/sdk';
+import { toRefs } from '@vueuse/shared';
```
Comment (csandru-plenty) [inline apps/web/composables/useActiveShippingCountries/useActiveShippingCountries.ts]: Nuxt handles auto-imports for it's own context, we can remove all manual imports (found in other files):
```import { toRefs } from '@vueuse/shared';```

File: apps/web/nuxt.config.ts (hunk 1 above)
Comment (maxiroellplenty) [reply apps/web/nuxt.config.ts]: done

File: apps/web/composables/useActiveShippingCountries/useActiveShippingCountries.ts (hunk 2 above)
Comment (maxiroellplenty) [reply apps/web/composables/useActiveShippingCountries/useActiveShippingCountries.ts]: done

**Review by ksted — APPROVED** [review]

**Review by FabianGerke — APPROVED** [review]

**Pull request merged**